lib/Frontend/nvqpp/ConvertDecl.cpp

```diff
@@ -298,12 +298,8 @@ bool QuakeBridgeVisitor::VisitVarDecl(clang::VarDecl *x) {
         return true;
 
       // Assign registerName
-      if (auto mx = initVec.getDefiningOp<quake::MxOp>())
-        mx->setAttr("registerName", builder.getStringAttr(x->getName()));
-      if (auto my = initVec.getDefiningOp<quake::MyOp>())
-        my->setAttr("registerName", builder.getStringAttr(x->getName()));
-      if (auto mz = initVec.getDefiningOp<quake::MzOp>())
-        mz->setAttr("registerName", builder.getStringAttr(x->getName()));
+      if (auto mz = initVec.getDefiningOp<quake::MeasurementInterface>())
```

Comment:
nit pick - I would change mz to measurment or something like that

Reply:
measurement will exceed 80 characters and wrap the line, so hopefully meas works for you?